apps/sensenet/src/components/view-controls/permission-view.tsx

```diff
@@ -42,8 +44,14 @@ const useStyles = makeStyles((theme: Theme) => {
     },
     title: {
       fontSize: '20px',
+      paddingRight: '10px',
+      width: `calc(100% - ${permissionActionbuttonsWidth}px)`,
```

Comment:
I think you could remove width and margin set, and get the same result without magic numbers.

Reply:
without these it seems like the screenshot:
![image](https://user-images.githubusercontent.com/22347393/95598186-ec6a0f80-0a4f-11eb-8d15-050578725bf5.png)